generator.yaml

```diff
@@ -15,6 +15,8 @@ operations:
   UpdateEndpoint:
     custom_implementation: customUpdateEndpoint
     set_output_custom_method_name: customUpdateEndpointSetOutput
+    override_values:
+      RetainAllVariantProperties: true
```

Comment:
This means that the user cannot customize this value correct  ?

Reply:
Correct